linode/manager
# chore: assets and utilities clean up

## Description 📝

- Clean up files in `packages/manager/src/assets/`  and `packages/manager/src/utilities/`
- Deleted util functions that were not used and that I don't think were worth keeping
- Removed some old unused assets
- Removed some default exports

## Preview 📷

- There should be no visual changes 🚨

## How to test 🧪

- Verify automated testing passes
- Do a quick once-over of the entire App and check for crashes and regressions
